crates/utils/src/settings/mod.rs

```diff
@@ -5,23 +5,21 @@ use crate::{
 };
 use anyhow::{anyhow, Context};
 use deser_hjson::from_str;
-use once_cell::sync::Lazy;
 use regex::{Regex, RegexBuilder};
-use std::{env, fs, io::Error, sync::RwLock};
+use std::{env, fs, io::Error};
 
 pub mod structs;
 
 static DEFAULT_CONFIG_FILE: &str = "config/config.hjson";
 
-static SETTINGS: Lazy<RwLock<Settings>> =
-  Lazy::new(|| RwLock::new(Settings::init().expect("Failed to load settings file")));
-static WEBFINGER_REGEX: Lazy<Regex> = Lazy::new(|| {
-  Regex::new(&format!(
+lazy_static! {
+  static ref SETTINGS: Settings = Settings::init().expect("Failed to load settings file");
+  static ref WEBFINGER_REGEX: Regex = Regex::new(&format!(
     "^acct:([a-zA-Z0-9_]{{3,}})@{}$",
     Settings::get().hostname
   ))
-  .expect("compile webfinger regex")
-});
+  .expect("compile webfinger regex");
+}
```

Comment:
Why switch this back to lazy_static? Using Lazy seems very much preferable, because it doesnt rely on macros.

Reply:
K, I'd assumed that lazy_static was preferable, but I'll switch back.